src/fluid.cpp

```diff
@@ -352,7 +352,7 @@ CDynamicAddress CFluid::GetAddressFromDigestSignature(std::string digestSignatur
 
     if (fInvalid) {
         LogPrintf("GenericVerifyInstruction(): Digest Signature Found Invalid, Signature: %s \n", digestSignature);
-        return false;
```

Comment:
This is temporary to allow building. This is from merging Amirs work, will await the fix for this from him.